standard/template/sections/clause_9_zarr_encoding_core.adoc

```diff
@@ -0,0 +1,160 @@
+
+=== Hierarchical Structure
+
+A dataset conforming to the unified data model is represented as a hierarchical structure of groups, variables (arrays), dimensions, and metadata. The dataset is rooted in a *top-level group*, which may contain:
+
+- Arrays representing coordinate or data variables
+- Child groups for modular organisation, including logical sub-collections or resolution levels
+- Metadata attributes at group and array levels
+
+Each group adheres to a consistent structure, allowing recursive composition. This reflects the CDM's use of *groups* and is supported by both Zarr v2 and v3 with differing implementations.
+
+
+[cols="1,2,2"]
+|===
+|Model Element |Zarr v2 Encoding |Zarr v3 Encoding
+
+|Root Dataset | Directory with `.zgroup` and `.zattrs` | Directory with `zarr.json`, with `node_type: group`
+
+|Child Group | Subdirectory with `.zgroup` and `.zattrs` | Subdirectory with `zarr.json`, with `node_type: group`
+
+|Array | Subdirectory with `.zarray` and `.zattrs` | Subdirectory with `zarr.json`, with `node_type: array`
+
+|Metadata Key | `.zattrs` file | `attributes` field in `zarr.json`
+|===
+
+Zarr v3 requires `zarr_format: 3` and stores all metadata (including user-defined attributes) in the `zarr.json` document. Each node includes a `node_type` field: either `"group"` or `"array"`.
+
+=== Dimensions
+
+Dimensions define the axes along which variables are indexed.
+
+- In Zarr v2, dimensions are inferred from array shape and declared in `_ARRAY_DIMENSIONS` within `.zattrs`.
+- In Zarr v3, dimensions are stored using the `dimension_names` field in `zarr.json`.
+
+Example for a 2D array with dimension names `["lat", "lon"]`:
+
+[source,json]
+----
+{
+  "zarr_format": 3,
+  "node_type": "array",
+  "shape": [180, 360],
+  "dimension_names": ["lat", "lon"],
+  ...
+}
+----
+
+=== Coordinate Variables
+
+Coordinate variables (excluding GeoTransform Coordinates) define the geospatial or temporal context of data. They are represented as named arrays with metadata attributes.
+
+Coordinate variables are represented as named 1D arrays aligned with corresponding dimensions.
```

Comment:
We should distinguish that coordinate variables are not part of the Zarr spec itself. It's from CDM.

Reply:
From my perspective its clear, as there is a mapping (section) for each component of the data model to Zarr. 
You will find the same sections to NetCDF, to GeoTiff, etc.